src/ecwrapper.cpp

```diff
@@ -177,8 +179,14 @@ bool CECKey::Recover(const uint256 &hash, const unsigned char *p64, int rec)
     if (rec<0 || rec>=3)
         return false;
     ECDSA_SIG *sig = ECDSA_SIG_new();
-    BN_bin2bn(&p64[0],  32, sig->r);
-    BN_bin2bn(&p64[32], 32, sig->s);
+    BIGNUM *sig_r, *sig_s;
```

Comment:
"Out, damned spot! Out, I say!—One, two. Why, then, ’tis time to do ’t. OpenSSL is murky!"